packages/material-ui/src/TablePagination/TablePagination.js

```diff
@@ -89,8 +89,8 @@ const TablePagination = React.forwardRef(function TablePagination(props, ref) {
     labelRowsPerPage = 'Rows per page:',
     nextIconButtonProps,
     nextIconButtonText = 'Next page',
-    onChangePage: onChangePageProp,
-    onPageChange: onPageChangeProp,
+    onChangePage,
+    onPageChange,
```

Comment:
We should make this optional prop too, no? https://github.com/mui-org/material-ui/blob/416742a8fddf03cffba2a68c90b63662bfb5f814/packages/material-ui/src/TablePagination/TablePagination.d.ts#L35

Reply:
Why is this relevant to prop destructuring?

Reply:
What I meant is that, in order to fix the issue, we need to make both props optional in this component too, as we made them in `TablePaginationActions`. Currently `onPageChange` is required.

Reply:
> What I meant is that, in order to fix the issue

What issue are you referring to? The original issue is fixed and the test suite is green. You're probably not talking about the runtime which is confusing since the conversation is targetted at the .js file.

Reply:
@mnajdova I would personally keep the types failing, to give a stronger signal to developers that they need to update their code. But no strong point of view.

Reply:
Agree the runtime issue is fixed, I thought that it's good chance to fix the changed interface as well, as it is a breaking change for the typescript users. Here is the discussion around it: https://github.com/mui-org/material-ui/pull/23789#issuecomment-876663828

Reply:
I don't understand how the types are failing. I only have the issue and test suite to go off. If you think there's a problem, please provide a concrete example.